gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -1615,6 +1655,43 @@ public String getAssignmentGradeComment(final String siteId, final long assignme
 		return null;
 	}
 
+	/**
+	 * Get the Excluded Status for a given student assignment grade
+	 *
+	 * @param assignmentId id of assignment
+	 * @param studentUuid uuid of student
+	 * @return the comment or null if none
+	 */
+	public Boolean getAssignmentExcludedStatus(final long assignmentId, final String studentUuid) {
+		final String siteId = getCurrentSiteId();
+		final Gradebook gradebook = getGradebook(siteId);
+		return gradebookService.getExculdedStatusForAssignmentByStudent(gradebook.getUid(), assignmentId, studentUuid);
+	}
+
+	/**
+	 * Set the Excluded Status for a given student assignment grade
+	 *
+	 * @param assignmentId id of assignment
+	 * @param studentUuid uuid of student
+	 */
+	public void setAssignmentExcludedStatus(final Long assignmentId, final String studentUuid) {
+		final String siteId = getCurrentSiteId();
+		final Gradebook gradebook = getGradebook(siteId);
+		gradebookService.toggleGradeRecordExcluded(gradebook.getUid(), assignmentId, studentUuid);
+	}
+
+	/**
+	 * Returns the assignment grade for a student
+	 *
+	 * @param assignmentId
+	 * @param studentUuid
+	 */
+	public double getAssignmentGradeByStudent (final Long assignmentId, final String studentUuid){
```

Comment:
To keep this class consistent, call this getGradeForStudent. Might be useful to use the GbGradeInfo type as well since if you use the existing gradebookService methods you'll get a GradeDefinition back, not a double. Consistency makes this easier to use.